cmd/skywire-cli/commands/config/root.go

```diff
@@ -41,7 +41,7 @@ var (
 	dmsgptywlPKs           string
 	surveywhitelistPks     string
 	routesetupnodePks      string
-	transportsetupnodePks  string
+	transportsetupPks  string
```

Comment:
Formatting. Camel case.

Reply:
some camels only have one hump

I had it different to differentiate it from what is in some instances the exact same name variable name in a struct which was not the same type of variable; namely here
```
			if services.TransportSetupPKs == nil {
```
I just find it confusing to have case-matching variable names which represent different types, here string and []cipher.Pubkey

Reply:
changed in latest commit